src/gui/qgisinterface.h

```diff
@@ -972,7 +972,7 @@ class GUI_EXPORT QgisInterface : public QObject
     virtual void removeDockWidget( QDockWidget *dockwidget ) = 0;
 
     //! Open layer properties dialog
```

Comment:
```suggestion
    /**
     * Opens layer properties dialog for the layer provided with \a l
     * A \a page to open can be specified (supported since QGIS 3.20).
     * Valid strings are [...], others may be available depending on the context.
     */
```

Reply:
I'd also note that the strings are subject to change without notice between QGIS versions, and that the tab names themselves aren't considered part of the stable API.

Reply:
Should be better now.